MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include LICENCE.rst
+global-include *.gro *.xml *.xsd *.top
```

Comment:
Are all of these files in a specific directory? I would prefer to specify the directories instead of the extensions. That way future trajectory files with different extensions won't be forgotten here.

For example, the `mol2` and `lammps` files in `tests/files` and the `mol2` files in `utils/files` won't be included. I'm not sure if that is needed to pass tests. https://github.com/mosdef-hub/gmso/tree/master/gmso/tests/files

Can you clarify the purpose of the files in `utils/files` vs. those in `tests/files`?

Reply:
And what about `log_config.json`? https://github.com/mosdef-hub/gmso/blob/master/gmso/utils/log_config.json

Reply:
I don't the purpose of log_config.json. I don't recall using a logger with `gmso` classes and what purpose it serves as it relates. Its a nice remainder because it is being used by `log.py` in utils. The test files are used for initializing objects/fixtures needed for testing and will be needed for tests to pass. @mosdef-hub/mosdef-maintainers could chime in for `utils/files`.

Reply:
I think we are better off including dirs. Thanks @bdice.